website/docs/r/container_app_environment.html.markdown

```diff
@@ -62,8 +62,22 @@ The following arguments are supported:
 
 * `log_analytics_workspace_id` - (Optional) The ID for the Log Analytics Workspace to link this Container Apps Managed Environment to. Changing this forces a new resource to be created.
 
+* `workload_profile` - (Optional) The profile of the workload to scope the container app execution. A `workload_profile` block as defined below.
+
 * `tags` - (Optional) A mapping of tags to assign to the resource.
 
+---
+
+A `workload_profile` block supports the following:
+
+* `name` - (Required) The name of the workload profile.
+
+* `workload_profile_type` - (Required) Workload profile type for the workloads to run on. 
```

Comment:
Can we add the valid values here too? 
```suggestion
* `workload_profile_type` - (Required) Workload profile type for the workloads to run on. Possible values include `D4`, `D8`, `D16`, `D32`, `E4`, `E8`, `E16`, and `E32`.
```
Can we also add a note here to inform users that the `Consumption` Workload Profile is always present and does not need to be defined.

Reply:
Sure. Will update the docs.